arm/Microsoft.Authorization/policyAssignments/readme.md

```diff
@@ -64,6 +67,27 @@ To deploy resource to a Resource Group, provide the `subscriptionId` and `resour
 
 > The `subscriptionId` is used to enable deployment to a Resource Group Scope, allowing the use of the `resourceGroup()` function from a Management Group Scope. [Additional Details](https://github.com/Azure/bicep/pull/1420).
 
+## Module Usage Guidance
+
+In general, most of the resources under the `Microsoft.Authorization` namespace allows deploying resources at multiple scopes (management groups, subscriptions, resource groups). The `deploy.bicep` root module is simply an orchestrator module that targets sub-modules for different scopes as seen in the parameter usage section. All sub-modules for this namespace have folders that represent the target scope. For example, if the orchestrator module in the [root](deploy.bicep) needs to target 'subscription' level scopes. It will look at the relative path ['/subscription/deploy.bicep'](./subscription/deploy.bicep) and use this sub-module for the actual deployment, while still passing the same parameters from the root module.
+
+The above method is useful when you want to use a single point to interact with the module but rely on parameter combinations to achieve the target scope. But what if you want to incorporate this module with other modules with lower scopes? This will not work as the [root](deploy.bicep) is defined at a higher scope (i.e. management group), hence the module can no longer be used. That is simply because you cannot have your own bicep file that has a target of subscription, and this root module is at a higher scope than that. This is the error that you can expect to face:
```

Comment:
```suggestion
The above method is useful when you want to use a single point to interact with the module but rely on parameter combinations to achieve the target scope. But what if you want to incorporate this module in other modules with lower scopes? This would force you to deploy the module in scope `managementGroup` regardless and further require you to provide its ID with it. If would not set the scope to management group, this would be the error that you can expect to face:
```

Reply:
My suggestion. If accepted, should be updated in the other modules too